src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -27,125 +27,250 @@
     </div>
   </div>
 
-  <div class="ev-card-panel">
-    <div class="bottom-card-container bottom-hr-line">
-      <strong class="fw-semibold content">
-        Please fill the following fields to submit
-      </strong>
-    </div>
-    <div class="row">
-      <div class="col-md-7 col-sm-7 col-xs-12">
-        <div class="exist-team-card">
-          <ul>
-            <b class="fw-bold content fs-14">Select phase:</b>
-            <app-selectphase [phases]="filteredPhases" [phaseSelectionType]="phaseSelectionType"
-              [phaseSelected]="phaseSelected()"></app-selectphase>
+  <div *ngIf="challenge.is_docker_based" class="ev-md-container ev-card-panel
+    card-bt-margin">
+    <div class="row row-lr-margin">
+      <div class="col-lg-12 col-md-12 col-sm-12 col-lr-pad">
+        <span class="fs-20 fw-light">
+          Submission instructions for docker image using evalai-cli
+        </span>
+        <ol>
+          <li class="content">Install evalai-cli<br>
+            <code>$ pip install evalai{{cliVersion}}</code>
+            <button id="checking" class="btn ev-btn-dark btn-waves-effect waves-dark grad-btn
+              code-btn" value='pip install "evalai{{cliVersion}}"'
+              (click)="copyTextToClipboard(install_cli)" #install_cli>
```

Comment:
Logically we still might hit the issue of having two strings, set the id to code tags and you should be able to get rid of value in button tag.

Reply:
done